ci/doc/maintenance.md

```diff
@@ -0,0 +1,13 @@
+If you have any question, please contact the below maintainers and committers.
+
+## Maintainers
+* Geunsik Lim (geunsik.lim@samsung.com)
+* MyungJoo Ham (myungjoo.ham@samsung.com)
+* Sewon Oh (sewon.oh@samsung.com)
+
+## Committers	
+* Jijoong Mon (jijoon.moon@samsung.com)
```

Comment:
Could you fix to Jijoong Moon? Thanks in advance.

Reply:
Fixed. Thanks.